src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/SecurityFactoryInterface.php

```diff
@@ -31,7 +46,18 @@ public function create(ContainerBuilder $container, $id, $config, $userProvider,
      */
     public function getPosition();
 
+    /**
+     * Defines the configuration key used to reference the provider
+     * in the firewall configuration.
+     * 
+     * @return string
```

Comment:
Ah, got me really confused there, I'd expect something like authenticator name or similar :) 

You can ignore my comment